app/views/admin/enterprises/form/_vouchers.html.haml

```diff
@@ -0,0 +1,33 @@
+.text-right
+  %a.button{ href: "#{new_admin_enterprise_voucher_path(@enterprise)}"}
+    = t('.add_new')
+%br
+
+- if @enterprise.vouchers.present?
+  %table
+    %thead
+      %tr
+        %th= t('.voucher_code')
+        %th= t('.rate')
+        %th= t('.label')
+        %th= t('.purpose')
+        %th= t('.expiry')
+        %th= t('.use_limit')
+        %th= t('.customers')
+        %th= t('.net_value')
+    %tbody
+      - @enterprise.vouchers.each do |voucher|
+        %tr
+          %td= voucher.code
+          %td= voucher.rate
```

Comment:
There's no `rate` on the voucher yet in this commit.

Reply:
True I am not sure what I did there. Anyway it has been replaced by `voucher.display_value` in this commit https://github.com/openfoodfoundation/openfoodnetwork/pull/10523/commits/e669bb06dc728e6ad9beba7bf9db23a35b4fe6c9